composer.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "owncloud/files_texteditor",
+  "config" : {
+    "platform": {
+      "php": "5.6.37"
+    }
+  },
+  "require": {
+    "christian-riesen/otp": "1.*",
```

Comment:
@dpakach I don't think these things are needed in files_texteditor. OTP and QR code stuff is used in the TOTP app, so these have got copy-pasted in here. An empty ```require`` section should work fine, like in ``password_policy``